src/cordova/build.action.mjs

```diff
@@ -33,35 +35,46 @@ export async function main(...parameters) {
   await runAction('cordova/setup', ...parameters);
 
   if (buildMode === 'debug') {
-    console.warn(`WARNING: building "${cordovaPlatform}" in [DEBUG] mode. Do not publish this build!!`);
+    console.warn(`WARNING: building "${outlinePlatform}" in [DEBUG] mode. Do not publish this build!!`);
   }
 
-  if (outlinePlatform === 'macos' || outlinePlatform === 'ios') {
-    const WORKSPACE_PATH = `${process.env.ROOT_DIR}/src/cordova/apple/${outlinePlatform}.xcworkspace`;
+  if (cordovaPlatform === 'osx' || cordovaPlatform === 'ios') {
```

Comment:
Why change to `cordovaPlatform`? That's just to interface with Cordova, it's better to stick to our `outlinePlatform` definition for consistency.
```suggestion
  if (outlinePlatform === 'macos' || outlinePlatform === 'ios') {
```